tools/msstatstmt/.shed.yml

```diff
@@ -0,0 +1,9 @@
+categories:
+  - Proteomics
+description: MSstatsTMT protein significance analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling
+long_description: |
+    MSstatsTMT provides statistical tools for detecting differentially abundant proteins in shot-gun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling.
+owner: galaxyp
+remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/msstatstmt
```

Comment:
there is a repo name missing isn't it?

Reply:
yes, indeed! I've used the msstats wrapper as a base.. it's missing there too.